README.md

```diff
@@ -1,5 +1,52 @@
 # ProActive Scheduler Client
 
-this module allows users to interact with a running ProActive Scheduler and Resource Manager.
+This module allows users to interact with a running ProActive Scheduler and Resource Manager.
 
-Learn more at [activeeon.com](https://www.activeeon.com)
+### Usage
+
+```
+import proactive
+
+print("Logging on proactive-server...")
+proactive_host = 'try.activeeon.com'
+proactive_port = '8080'
+proactive_url  = "http://"+proactive_host+":"+proactive_port
+print("Connecting on: " + proactive_url)
+gateway = proactive.ProActiveGateway(proactive_url)
+
+gateway.connect("bobot", "proactive")
```

Comment:
@andrewssobral I think it is a very bad idea to hardcode these credentials here!
Maybe you can leave a comment and ask the user to use his paXXXX credentials obtained upon signing up.

Reply:
@amouhoub fixed, thank you for the remark